BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -14,9 +14,27 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define RSA_KEY_VB_SIGNATURE     SIGNATURE_32 ('$', '_', 'V', 'B')
 #define RSA_KEY_IPP_SIGNATURE    SIGNATURE_32 ('$', 'I', 'P', 'P')
+#define PUBKEY_IDENTIFIER        SIGNATURE_32 ('P', 'U', 'B', 'K')
+#define SIGNATURE_IDENTIFIER     SIGNATURE_32 ('S', 'I', 'G', 'N')
+
 
 #define SIG_TYPE_RSA2048SHA256   0
+#define SIG_TYPE_RSA3072SHA384   1
+
+typedef UINT8 SIGN_TYPE;
+#define SIGNING_TYPE_RSA_PKCS_1_5     1
+#define SIGNING_TYPE_RSA_PSS          2
+
+typedef UINT8 KEY_TYPE;
+#define KEY_TYPE_RSA                  1
+
+#define RSA2048_NUMBYTES              256
+#define RSA3072_NUMBYTES              384
+

```

Comment:
RSA2048NUMBYTES  and RSA3072NUMBYTES are defined in line 50. should we only use one of them?

Reply:
RSA2048NUMBYTES and RSA3072NUMBYTES  are duplicated. I would removed in subsequent patch.
RSA2048_NUMBYTES and RSA3072 NUMBYTES defined above. We currently use RSA2048_NUMBYTES.

Reply:
It is fine to me to remove one of them later.